pop-api/README.md

```diff
@@ -70,4 +70,4 @@ So we have covered how the ink! Pop API library calls the chain extension. But w
 
 Chain extensions "extend" a runtime. We can find the `PopApiExtension` chain extension in [extension.rs](../runtime/devnet/src/extensions.rs). The `PopApiExtension` chain extension is matching based on the function IDs that we defined on the ink! side of the Pop API. The chain extension here will execute the appropriate functions e.g. `dispatch` or `read_state`. These functions are defined in this file as well and interact directly with the Pop Network runtime.
 
-If you would like to see the whole flow, checkout the end-to-end tests in [extensions.rs](../runtime/devnet/src/extensions.rs) file e.g. `dispatch_nfts_mint_from_contract_works()`
```

Comment:
Please remove, this change is not relevant to the PR and only creates distraction for the reviewer

Reply:
Thanks! Will address